ansible/roles/jenkins-worker/vars/main.yml

```diff
@@ -87,6 +87,10 @@ needs_monit: [
   'debian7',
   ]
 
+archtype: {
+  macos11.0: 'arm64'
```

Comment:
Is there not going to be any Intel testing on macOS 11?

Reply:
Currently no, we dont have the machines for it.

Reply:
@AshCripps we probably add a task to add macOS 11 machines for intel, I believe we have enough capacity to do that right?